packages/docs-ui/src/services/clipboard/clipboard.service.ts

```diff
@@ -132,6 +132,11 @@ export class DocClipboardService extends Disposable implements IDocClipboardServ
     async legacyPaste(html?: string, text?: string): Promise<boolean> {
         const partDocData = this._genDocDataFromHtmlAndText(html, text);
 
+        // Paste in sheet editing mode without paste style
+        if (this._univerInstanceService.getCurrentUnitForType(UniverInstanceType.UNIVER_DOC)?.getUnitId() === DOCS_NORMAL_EDITOR_UNIT_ID_KEY) {
+            delete partDocData.body?.textRuns;
```

Comment:
prefer set the `testRuns` to empty array.

It's better to add a comment on why this logic was added, so that it can be removed when refactoring